package.json

```diff
@@ -74,4 +74,4 @@
   "engines": {
     "yarn": ">= 1.0.0"
   }
-}
+}
```

Comment:
Thanks for the great work @anirudhprabhakaran3, as @cesswairimu mentioned, you have a change in the package.json file that is not required in the changes. Is this extra bracket required in this issue?